lib/new_post_manager.rb

```diff
@@ -73,6 +73,7 @@ def self.user_needs_approval?(manager)
 
     (user.trust_level <= TrustLevel.levels[:basic] && user.post_count < SiteSetting.approve_post_count) ||
     (user.trust_level < SiteSetting.approve_unless_trust_level.to_i) ||
+    (manager.args[:title].present? && user.trust_level < SiteSetting.approve_new_topics_unless_trust_level.to_i) ||
```

Comment:
Tests added (and they helped me find another needed fix). It's my first rails test, so please check carefully.